diff --git a/vmdebootstrap-generic-qcow2/common.mk b/vmdebootstrap-generic-qcow2/common.mk
index 3bf6c50..2ec79ad 100644
--- a/vmdebootstrap-generic-qcow2/common.mk
+++ b/vmdebootstrap-generic-qcow2/common.mk
@@ -16,7 +16,7 @@ HTTP_PROXY_SETTING := $(shell nc -z 127.0.0.1 3142 && echo 'http_proxy=http://12
 		$(VMDEBOOTSTRAP_OPTIONS) \
 		$(EXTRA_VMDEBOOTSTRAP_OPTIONS) \
 		--verbose \
-		--size 10G \
+		--size 20G \
 		--distribution $* \
 		--grub \
 		--arch $(ARCH) \
diff --git a/vmdebootstrap-libvirt-vagrant/metadata.json.in b/vmdebootstrap-libvirt-vagrant/metadata.json.in
index e2e3fef..80a3a77 100644
--- a/vmdebootstrap-libvirt-vagrant/metadata.json.in
+++ b/vmdebootstrap-libvirt-vagrant/metadata.json.in
@@ -1,5 +1,5 @@
 {
   "provider"     : "libvirt",
   "format"       : "qcow2",
-  "virtual_size" : 10
+  "virtual_size" : 20
 }
